test_unstructured_inference/inference/test_layout.py

```diff
@@ -267,3 +267,8 @@ def mock_get_elements(self, *args, **kwargs):
         .elements
     )
     assert elements[0] == mock_page_layout
+
+
+def test_from_image_file_raises_with_empty_fn():
+    with pytest.raises(FileNotFoundError):
```

Comment:
Should we also have a test for the case where the user passes in a directory?

Reply:
Case added in [ff0f5db](https://github.com/Unstructured-IO/unstructured-inference/pull/19/commits/ff0f5db55d0482be82f3ae71422bffde5bad163e).